frank-doc-doclet/src/main/java/org/frankframework/frankdoc/FrankDocJsonFactory.java

```diff
@@ -239,6 +239,16 @@ private JsonObject getElement(FrankElement frankElement) throws JsonException {
 			}
 			result.add("tags", b.build());
 		}
+		if(! frankElement.getLabels().isEmpty()) {
```

Comment:
Ik denk dat het handig is om in de json naast de labels van elk FrankElement ook een globaal lijstje te hebben van alle labels die er zijn, met een lijstje van hun waarden. Daar kunnen dan in de frankdoc-webapp bijvoorbeeld dropdowns uit gegenereerd worden.

Reply:
Gedaan.